src/puppetlabs/trapperkeeper/services/webserver/jetty9_core.clj

```diff
@@ -361,6 +364,7 @@
                                 :client-auth     (:client-auth https)
                                 :ssl-crl-path    (:ssl-crl-path https)
                                 :cipher-suites   (:cipher-suites https)
+                                :allow-renegotiation (:allow-renegotiation https)
```

Comment:
Nice to have: using [`select-keys`](https://clojuredocs.org/clojure.core/select-keys) instead of manual map construction.

Reply:
My Clojure skills are rudimentary at best so my changes are not the most elegant but would be happy to adjust once I understand/figure out the above construct.

Reply:
Here we're just building a smaller map to pass into `ssl-context-factory`, using as far as I can tell, a subset of the keys (without changing any names) from the `https` map. Select-keys will do exactly that for us like:
```clj
  (let [ssl-ctxt-factory (ssl-context-factory
                           (select-keys https
                                       [:keystore-config :client-auth
                                        :ssl-crl-path :cipher-suites
                                        :allow-renegotiation]))])
```

```